cas/cas.h

```diff
@@ -338,6 +338,12 @@ enum m0_cas_op_flags {
 	 * In other words, it might be used in degraded mode.
 	 */
 	COF_SHOW_DEAD = 1 << 10,
+	/**
+	 * NO DTM is needed for this operation.
+	 * FIXME: do we really need to pass this flag to cas service?
```

Comment:
@nikitadanilov ?

Reply:
I have removed this comments from the code.
I think the answer is yes.